# DSC180A MA5

## Gurman Dhaliwal (gdhaliwa@ucsd.edu)
### Section A02. Mentor: Dr. Tiffany Amariuta

1. What is the most interesting topic covered in your domain this quarter?

I think the most interesting topic was linkage disequilibrium, which refers to a phenomenon where a group of single nucleotide polymorphism (SNPs) in a particular region are correlated with a certain gene expression, but only a few of these SNPs have a causal impact on the gene expression. The processes to identify the causal SNPs, including pruning and thresholding, are interesting. It was also interesting to learn that the diseases with more SNPs with more stringent thresholds indicate a higher complexity of the disease since that disease is caused by multiple SNPs. 

2. Describe a potential investigation you would like to pursue for your Quarter 2 Project.
   
A potential investigation that I would like to pursue for my Quarter 2 project is a rare variant analysis where we investigate the impact of the rare genetic variants that might be underrepresented but still significant on the gene expression in this dataset. A drawback of this is that this dataset is relatively small, so it may be harder to identify significant rare variants. Another potential investigation is to do fine mapping to identify which SNPs have a causal relationship with certain traits. 

3. What is a potential change you’d make to the approach taken in your current Quarter 1 Project?

Our quarter 1 project was pre-decided for everyone. Nevertheless, one potential change I’d make to the approach taken would be to have more individuals for part 2 of the dataset whose data that we could plot to assess where they fell on the distribution. It would also be great if we had some grounded results (from 23AndMe and the individuals’ consent) to compare our placement of the individuals on a particular gene expression’s distribution with. This would help us understand how far off our models are and if they work better for certain gene expressions and we could potentially experiment with more complex models for better results. 

4. What other techniques would you be interested in using in your project?

Some other techniques that I would be interested in using in my project are potentially doing more linkage disequilibrium analysis and examining whether the non-random associations between different variants have any insights on complex traits. It would also be interesting to learn more about haplotype mapping and if the combination of the haplotypes or alleles has any significance in gene expression association. In terms of technical techniques, I would like to try some of the packages and techniques that I’ve been using in Python in R. I would also like to use the UC Santa Cruz genome website to extract some relevant information and visualization as well get some more background research for the quarter 2 project. Additionally, I would like to better learn how to integrate LDs into the LocusZoom online interactive software to make the graphs more informative. 



